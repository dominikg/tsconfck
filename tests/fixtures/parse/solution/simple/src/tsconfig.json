{
  "include": ["./foo.ts"],
  "extends": "../tsconfig.base",
  "compilerOptions": {
    "strict": true
  }
}
