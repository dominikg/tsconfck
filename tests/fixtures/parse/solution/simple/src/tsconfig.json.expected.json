{
  "include": [
    "./foo.ts"
  ],
  "extends": "../tsconfig.base",
  "compilerOptions": {
    "strict": true,
    "composite": true,
    "strictNullChecks": true
  }
}
